exonum/src/node/requests.rs

```diff
@@ -74,7 +74,7 @@ impl NodeHandler {
         trace!("HANDLE TRANSACTIONS REQUEST");
         let snapshot = self.blockchain.snapshot();
         let schema = Schema::new(&snapshot);
-        const TX_HEADER: u32 = 8 + 2;
+        const TX_HEADER: u32 = 4; // service_id + message_id
```

Comment:
Hm... I just noticed that `exonum/src/sandbox/consensus/transactions.rs` contains exactly the same definition. Aren't these the same? If so then I think it should be defined in one place and imported from there.